### PR TITLE
Add awesomify function

### PR DESCRIPTION
**Pull request by diddledan**
This PR adds the awesomify function from [diddledan/awesomify](https://github.com/diddledan/awesomify).

Signed-off-by: Daniel Llewellyn <daniel@bowlhat.net>

### CODE REVIEW

File: store.json [hunk 1]
```diff
@@ -221,5 +221,18 @@
     "labels": {
       "com.openfaas.ui.ext": "jpg"
     }
+  },
+  {
+    "title": "Awesomify",
+    "description": "Make anyone or anything awesome!",
```
Comment (alexellis) [inline store.json]: I'm not sure that this meme or tool is widely-known. It may benefit from being more descriptive in the entry.

Comment (diddledan) [reply store.json]: I've switched the description to a much longer and more explanatory one. Can you give it another look to see if it's more suitable? Thanks.

**Comment by alexellis** [comment]
Thanks for updating. I think the description may be OTT now, it may not render correctly in the UI.

Alex

**Comment by alexellis** [comment]
@diddledan thanks for the submission, please could you:

* shorten the description
* submit the same into `functions.json`
* alter `awesomify.yml` to `stack.yml` in your repo
* See how we are downloading the multi-arch watchdog and update to that version (it now uses multi-stage builds)
* update `alpine:3.8` to `alpine:3.9`

Thanks a lot, happy #FaaSFriday !

**Pull request merged**